addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -8,6 +8,10 @@
       "playSound": false
     }
   },
+  "scripts": {
+    "start": "ng server",
+    "postinstall": "typings install --ambient"
```

Comment:
Does not remove the typings removed from typings.json.

I filed https://github.com/typings/typings/issues/170 to do just that.